toolkit/docs/building/prerequisites-ubuntu.md

```diff
@@ -3,26 +3,26 @@
 
 ## Requirements were validated on `Ubuntu 18.04`
 
-Requirements for building images with a toolkit:
+This page lists host machine requirements for building with the CBL-Mariner toolkit. They cover building the toolchain, packages, and images on an Ubuntu 18.04 host.
 
 ```bash
 # Add a backports repo in order to install the latest version of Go.
 sudo add-apt-repository ppa:longsleep/golang-backports
 sudo apt-get update
 
 # Install required dependencies.
-sudo apt -y install git make tar wget curl rpm qemu-utils golang-1.17-go genisoimage python-minimal bison gawk parted
+sudo apt -y install curl gawk genisoimage git golang-1.17-go make parted qemu-utils rpm tar wget
```

Comment:
Should we add `createrepo` here as well ?

Reply:
I don't believe createrepo is needed externally - just for our internal build system.